docs/learn/voting-dapp/voting-dapp.mdx

```diff
@@ -0,0 +1,679 @@
+Blockchain and web3 have changed our understanding of transparency, trust and decentralization. One of the most incredible use-case for this technology is the development of web3 dApps(Decentralized Applications) that offers solutions that are secure, verifiable and tamper-proof solutions to our everyday challenges. Among all the use-cases, decentralized voting stands out. Decentralized technologies remove middlemen and ensure transparent vote counts, this voting dApp has the potential to change governance system while enhancing the democratic process.
+
+In this tutorial, we will build a simple voting dapp on linea, a zkEVM layer 2 blockchain solution. You will learn how to write, deploy and interact with a smart contract which is designed for decentralized voting process. By the end of this tutorial, you’ll have the solid understanding of the tools and features provided by linea for decentralized applications. 
+
+
+
+## Architecture
+
+The voting dApp has three main components: smart contracts, smart contracts integration and the frontend interface.
```

Comment:
```suggestion
The voting dapp has three main components: smart contracts, smart contracts integration and the frontend interface.
```

Please use `dapp` throughout.

"smart contracts integration" also needs some attention but I'm not entirely clear what you're referring to, so unsure what to suggest. What does this statement mean?